yocto_dobi_gen.yaml

```diff
@@ -0,0 +1,69 @@
+#
+# This generates the yocto 
+#
+
+meta:
+  project: ci.os.lmp
+
+# ===================================================
+# mount points
+# ===================================================
+mount=source-dir:
+  bind: yocto
+  path: /data
+  read-only: false
+
+# ===================================================
+# images
+# ===================================================
+
+image=image-jinja:
+  image: pinterb/jinja2
+  tags: ["0.0.16"]
+  pull: once
+
+# ===================================================
+# jobs
+# ===================================================
+
+job=template: &Template
+  use: image-jinja
+  mounts:
+    - source-dir
+  sources: yocto/dobi_j2.yaml
+
+job=cpu01-bringup:
+  <<: *Template
+  artifact: yocto/.generated_cpu01-bringup.yaml
+  command: image_name='cpu01-bringup' image_pattern='moducop-cpu01/*Bringup*CI.OS.*.mender'
+  env: 
+    - "PUID={user.uid}"
+    - "PGID={user.gid}"
+    - "TEMPLATE=dobi_j2.yaml"
```

Comment:
jinja templates usually named by filename.ening.j2 -> dobi.yaml.j2

Reply:
Yes, but then, the vscode syntax highligting for YAML doesn't work anymore. Do you have an idea how to solve that?

Reply:
Leave it